### ISSUE TITLE
CustomEvent polyfill problem in SSR mode

### ISSUE THREAD

**Issue by eleon-sqli** [reactstrap/reactstrap#1858]
- components: `polyfill.js`
- reactstrap version `#8.5.0`

### What is happening?

Cannot build a Next.js project in production mode with the last version of reactstrp.

### What should be happening?

No problem when using previous reactstrap versions.

### Steps to reproduce issue

Just upgrade reactstrap to #8.5.0 and build

### Error message in console

```
Build error occurred
ReferenceError: window is not defined
    at <project root>/node_modules/reactstrap/lib/polyfill.js:4:3
```

**Comment by iamandrewluca** [comment]
Fixed in #1855

**Comment by eleon-sqli** [comment]
Thanks Andrew :)
